permissions.md

```diff
@@ -1,13 +1,20 @@
 # Permissions
 
-<!-- > ### In 4.2 permissions are changing! Please read the `Permissions (4.2)` page for more info! -->
+## Version 4.2 permissions changes
+In version 4.2 we have changed the permissions system to be, we believe, better. Here are some changes:
+* There is no longer a default group. We have replaced this with a dynamic system that means that users get default permissions without a group. To list all of the default permissions, please refer to the `_perms list` command!
+* We have introduced negation permissions meaning by putting a `-` before a permission will **deny** users access to the permission. This works in terms of group hierarchy; groups at the top will override groups at the bottom. User permissions will also immediately override group permissions.
+* As mentioned before, groups now follow a hierarchy (Groups at the top being more important). To allow for this order to be changed we have introduced the command `_perms group <group> move <position>` to move the group up and down the chain. This order can bee seen by doing `_perms groups`!
+* We have finally added the ability to do full and accurate permission validation meaning you will never be mistake on a permission again!
+
 
 ## Important Notes
-* All FlareBot permissions follow a pattern of `flarebot.command` and `flarebot.command.sub-command` if applicable. 
+* All FlareBot permissions follow a pattern of `flarebot.command` and `flarebot.command.sub-command` if applicable.
+* All the permissions can be listed with `_perms list`. This command also tells you if the permissions are default or not.
 * Everyone with `Administrator` Discord permission will have access to **all** commands.
-* Everyone already has a group before you do anything, the group is called `Default`.
+* Everyone already has a select group of permissions to start off with. These are listed in the "Default" section of `_perms list`! To remove these permissions simply add the permission with a `-` before it to a user! (There should be no space between the `-` and the permission i.e. `-flarebot.stop`)
```

Comment:
You said you can ass this to a user but not a group.
Also stop isn't default, use something like `play` as the example.